### ISSUE TITLE
focusedWindow/frontmostWindow returning nil

### ISSUE THREAD

**Issue by trodrigues** [Hammerspoon/hammerspoon#1911]
I believe this started happening on version 0.9.67 or later, but when calling `hs.window.focusedWindow()` I always get `nil`. `frontmostWindow` does the same thing.

I'm still on macOS Sierra so I wonder if this could be somewhat related.

**Comment by cmsj** [comment]
Could you double check that Hammerspoon still has Accessibility permissions in the System Preferences?

**Comment by trodrigues** [comment]
Yep, it does. And usually when it loses those permissions it asks me to reactivate them again.

However, for some reason I appear to not be able to reproduce this issue anymore. When I first reported it I restarted Hammerspoon multiple times and tweaked my config to try and figure out what was going, and even after a whole system restart it persisted.

I can't really tell what (if anything) I've done to fix this. I believe there was a minor macOS update ever since I first reported this so I wonder if that had any relation but I can't be sure.

As such I'll close this and report back if it ever happens again and I can get more information.
